PlatformAuthentication/explainer.md

```diff
@@ -0,0 +1,324 @@
+# Allow JS App to directly interact with platform broker 
```

Comment:
Let's give this a clearer, more easily referenceable name. How about "Enterprise Platform Authentication Broker API"?